content/en/platform/corda/5.1/application-networks/creating/members/reregister.md

```diff
@@ -0,0 +1,208 @@
+     "corda.ledger.keys.0.signature.spec": "SHA256withECDSA",
+     "corda.endpoints.0.connectionURL": "https://'$P2P_GATEWAY_HOST':'$P2P_GATEWAY_PORT'",
+     "corda.endpoints.0.protocolVersion": "1",
+     "ext.sample": "apple"
+   }'
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+   ```shell
+   $REGISTRATION_CONTEXT = @{
+     'corda.session.keys.0.id' =  $SESSION_KEY_ID
+     'corda.session.keys.0.signature.spec' = "SHA256withECDSA"
+     'corda.ledger.keys.0.id' = $LEDGER_KEY_ID
+     'corda.ledger.keys.0.signature.spec' = "SHA256withECDSA"
+     'corda.endpoints.0.connectionURL' = "https://$P2P_GATEWAY_HOST`:$P2P_GATEWAY_PORT"
+     'corda.endpoints.0.protocolVersion' = "1",
+     'ext.sample' = "apple"
+   }
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
+3. Optional: Include serial number in the registration context.
+
+   The `corda.serial` Corda platform property is embedded in the `MemberInfo` of all members. It acts as the `MemberInfo`'s version,
+   and is incremented by 1 after each registration.
+
+   This serial number may be optionally included in the registration context to specify which version of the `MemberInfo` was
+   intended to be updated. If no serial number is provided while re-registering, the platform will use the member's latest
+   serial number by default. Only the latest `MemberInfo` version may be updated - requests with an older serial number are declined.
+   It is recommended to provide the serial number to avoid unintentional updates, in case you have an outdated version of the `MemberInfo`.
+
+   You can retrieve the serial number from the MGM-provided context of the `MemberInfo` by performing a member lookup.
+
+   {{< note >}}
+   For first-time registration the serial number should be 0, if provided.
+   {{< /note >}}
+
+   {{< tabs >}}
+   {{% tab name="Bash"%}}
+   ```bash
+   export REGISTRATION_CONTEXT='{
+     "corda.session.keys.0.id": "'$SESSION_KEY_ID'",
+     "corda.session.keys.0.signature.spec": "SHA256withECDSA",
+     "corda.ledger.keys.0.id": "'$LEDGER_KEY_ID'",
+     "corda.ledger.keys.0.signature.spec": "SHA256withECDSA",
+     "corda.endpoints.0.connectionURL": "https://'$P2P_GATEWAY_HOST':'$P2P_GATEWAY_PORT'",
+     "corda.endpoints.0.protocolVersion": "1",
+     "ext.sample": "apple",
+     "corda.serial": "1"
+   }'
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+   ```shell
+   $REGISTRATION_CONTEXT = @{
+     'corda.session.keys.0.id' =  $SESSION_KEY_ID
+     'corda.session.keys.0.signature.spec' = "SHA256withECDSA"
+     'corda.ledger.keys.0.id' = $LEDGER_KEY_ID
+     'corda.ledger.keys.0.signature.spec' = "SHA256withECDSA"
+     'corda.endpoints.0.connectionURL' = "https://$P2P_GATEWAY_HOST`:$P2P_GATEWAY_PORT"
+     'corda.endpoints.0.protocolVersion' = "1",
+     'ext.sample' = "apple",
+     'corda.serial' = "1"
+   }
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
+4. Send a re-registration request using the common registration/re-registration endpoint.
```

Comment:
Might be nice to name the endpoint and/or link to it in the API docs.

Reply:
@YashNabar Is that the correct endpoint? https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/openapi.html#tag/Member-Registration-API/operation/post_membership__holdingidentityshorthash_

Reply:
Yes, that is correct @AnnaK-R3